Assignments/Sorting/SelectionSort.playground/Pages/Selection Sort.xcplaygroundpage/Contents.swift

```diff
@@ -11,17 +11,35 @@ Better define an inner function to find the next minimum in the array.
 
 func selectionSort(array: [Int]) -> [Int] {
     // You may declare array argument with var keyword so that it is copied.
-
-    return array
+    var anArray = array
+    var index = 0
+    while (!anArray.isSorted()) {
```

Comment:
Hi, will try to modify my response later tomorrow..now it is almost midnight here :)